clients/client-java/src/main/java/com/datastrato/gravitino/client/BaseSchemaCatalog.java

```diff
@@ -81,7 +81,7 @@ public NameIdentifier[] listSchemas() throws NoSuchCatalogException {
             ErrorHandlers.schemaErrorHandler());
     resp.validate();
 
-    return resp.identifiers();
+    return Arrays.stream(resp.identifiers()).map(NameIdentifier::name).toArray(String[]::new);
```

Comment:
So, will the server side of this interface remain the same?

Reply:
Can you respond to it?

Reply:
Right, server side won't change; The "SupportsSchemas" has two version, one is for client, the other is for server side. This change has already been made in the main branch.